docs/discovery/xapi-problem-check.json

```diff
@@ -0,0 +1,106 @@
+{
+	"_comment": "This xAPI statement attempts to fully map all the data represented in the canonical sample event here https://edx.readthedocs.io/projects/devdata/en/latest/internal_data_formats/tracking_logs/sample_events.html",
+	"id": "8f87ccde-bb56-4c2e-ab83-44982ef22df0",
+	"actor": {
+		"objectType": "Agent",
+		"account": {
+			"homepage": "https://openedx.org/",
+			"name": "b5059046-d6db-11ec-8490-381428857bd9"
+		}
+	},
+	"verb": {
+		"_comment": "The tracking event type this sample corresponds to is 'problem_check'",
+		"id": "http://adlnet.gov/expapi/verbs/answered",
+		"display": {
+			"en-US": "answered"
+		}
+	},
+	"object": {
+		"objectType": "Activity",
+		"_comment": "This URL is a placeholder utilizing the 'usage_key'.  Should be instance specific, but perhaps transferable across domains?",
+		"id": "https://openedx.org/block-v1:edX+DemoX+Demo_Course+type@problem+block@29c5cbd840324d94be8ba51db1864277",
+		"definition": {
+			"name": {
+				"en-US": "Which of the following is a fruit?"
+			},
+			"description": {
+				"en-US": "A diagnostic assessment related to fruit literacy."
+			},
+			"type": "http://adlnet.gov/expapi/activities/assessment"
+		},
+		"_comment1": "The choicehint content feels most natural here in the activity definition rather than the response.  It is packed into the user response in the current event format.",
+		"extensions": {
+			"https://openedx.org/xapi/choicehint": "<choicehint selected=\"true\">You are correct that an apple is a fruit because it is the fertilized ovary that comes from an apple tree and contains seeds.</choicehint>\n      <choicehint selected=\"false\">Remember that an apple is also a fruit.</choicehint>\n"
```

Comment:
I did not see an extensions that seemed appropriate.  I wonder if this data is just too unnatural to include in an xAPI statement.

@ormsbee do you have any specific recollection as to why this data was included in native Open edX events?

Reply:
I don't have a recollection of it, but my guess would be because it matters for later analysis whether a student got the problem without hints, whether the hints helped them to get the correct answer, and which hint did so.